how-to-build-deep-learning-model-using-tensorflow-with-python/Index.md

```diff
@@ -0,0 +1,117 @@
+# How To Build Deep learning Models Using Tensorflow With Python
+
+## Prerequisites
+To get along, you should ;
+- Have a little knowledge about deep learning models and machine learning
+- Be farmiliar with deep learning basics and how to go about it, you refer to this article [Introduction to Deep Learning](https://www.section.io/engineering-education/introduction-to-deep-learning/) to learn more about deep learning
+## Introduction
+Giving a look at the world today, you’ll realize that many applications that are used on our daily lives have been built comprising deep learning. 
+Deep learning is a subset of machine learning that mimics the neuron of the human brain, 
+it allows the computer to be able to do human tasks by imitating the human thinking process.
+Deep Learning Models create a network that resembles biological nervous system.Frameworks used to build these deep learning models include caffee, Microsoft cognitive tool, MXNET, Keras, DeepLearning4j and Tensorflow. 
+Keras and Tensorflow are the mostly used frameworks.
+
+### *Table of content*
+- [Importance of deep learning](#importance-of-deep-learning)
+- [What is Tensorflow?](#what-is-tensorflow?)
+- [Merits and demerits of tensorflow](#merits-and-demerits-of-tensorflow)
+  - [*Merits*](#*merits*)
+   - [*Demerits*](#*demerits*)
+- [Steps of Tensorflow algorithm](#steps-of-tensorflow-algorithm)
+- [Conclusion](#conclusion)
+
+#### **Importance of deep learning**
+It has the capacity to process several features hence making it best in operating with unstructured data. Deep learning technology is able to lead to a dramatic development in our day-to- day quality of life by helping us predict, plan, make and apply better decision.
+
+With knowledge in modelling deep learning neural networks and a better understanding in probabilistic methods one should be able start his/her career in deep learning with ease.
+Some of the modelling deep learning neural networks that will may be of great help in your career include ADAM, CNN, LSTM, Dropout, RNN, etc. 
+
+#### __What is Tensorflow?__
+TensorFlow <span style="color:green">is a free, open-source machine learning software that can be used from start to end.
+</span> It has a large, flexible set of operators, libraries, and community resources that allow academics to advance the state-of-the-art in machine learning and developers to quickly construct and deploy machine learning applications.
+Tensorflow was developed in 2015 by google.
+
+#### **Merits and Demerits of TensorFlow**
+The following are some of the advantages of tensorflow ;
+
+#### Merits
+- Tensorflow is very compatible with a lot of languages like;  javascript, python, c#, swift, c++, and ruby hence the user may be choose any language to use tensorflow.
+- With its graphical approach, TensorFlow offers a superior way of visualizing data hence with the aid of TensorBoard, it is possible to easily debug nodes.

```

Comment:
This is factually incorrect. The underlying structure is a graph. It is not the approach.